__tests__/app.test.js

```diff
@@ -183,5 +184,127 @@ describe("GET /api/users", () => {
         });
       });
   });
-  
+});
+
+describe.only("GET /api/articles", () => {
+  test("Status 200: return with array of objects ommiting the article body & should be sorted in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).toBe(12);
+
+        body.articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              title: expect.any(String),
+              topic: expect.any(String),
+              author: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              article_id: expect.any(Number),
+            })
+          );
+        });
+        expect(body.articles).toEqual([
```

Comment:
It might be easier to use jest-sorted to check that they are in descending order, instead of having to write the whole array out. 
If you were to change the test data, this test will break. 
With jest sorted, it wont